extensions/resource-deployment/src/services/tools/azCLITool.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+import { ToolType, ITool, ToolInstallationStatus } from '../../interfaces';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+export class AzCLITool implements ITool {
```

Comment:
nitpick: could we just use `Cli` instead of `CLI`? The partial uppercase class names always look weird to me.

Reply:
sure, if that is the common practice.